### PR TITLE
fix(glue): Make targetTable optional

### PR DESCRIPTION
**Pull request by moomindani**
This PR makes `targetTable` parameter optional when defining Glue Data Quality Rulesets.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by moomindani** [comment]
Exemption Request: This is a minor fix to make one parameter optional. No need to add integration test.

**Review by daschaa — CHANGES_REQUESTED** [review]
Even if this is a minor change, I would prefer to have an integration test for this change. Is there a strong reason to not do an integration test for this?

File: packages/@aws-cdk/aws-glue-alpha/lib/data-quality-ruleset.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export interface DataQualityRulesetProps {
    * The target table of the ruleset
    * @attribute
```
Comment (lpizzinidev) [inline packages/@aws-cdk/aws-glue-alpha/lib/data-quality-ruleset.ts:76]: The `@default` value or behavior for the optional variable should be specified.

**Review by lpizzinidev — CHANGES_REQUESTED** [review]
I agree with @daschaa that an integration test should be provided to verify that a `DataQualityRuleset` without a `targetTable` deploys correctly.
You should be able to use [this](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-glue-alpha/test/integ.data-quality-ruleset.ts) test as a blueprint.

**Comment by moomindani** [comment]
Sure I will do that.

**Comment by moomindani** [comment]
This is waiting for one dependency being solved. Will work on it when it becomes actionable.

**Pull request closed without merging**